src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -934,22 +942,41 @@ class ongoingTasksStats extends viewModelBase {
         });
 
         this.subscriptionData.forEach(subscriptionTask => {
+            const subscriptionName = subscriptionTask.TaskName;
+            let subscription = this.subscriptionToWorkers.get(subscriptionName);
+            
+            if (subscription) {
+                subscriptionTask.ConnectionPerformance.forEach(connection => {
+                    subscription.add(connection.WorkerId);
+                })
+            } else {
+                const workerIds = subscriptionTask.ConnectionPerformance.map(x => x.WorkerId);
+                this.subscriptionToWorkers.set(subscriptionName, new Set<string>());
+                subscription = this.subscriptionToWorkers.get(subscriptionName);
+                workerIds.forEach(x => subscription.add(x));
```

Comment:
```
const workerIds = subscriptionTask.ConnectionPerformance.map(x => x.WorkerId);
this.subscriptionToWorkers.set(subscriptionName, new Set<string>([workerIds]));
```

instead of those 4 lines

Reply:
done